std_msgs/Bool up
std_msgs/Bool down
std_msgs/Bool left
std_msgs/Bool right